src/modules/iotjs_module_adc.c

```diff
@@ -47,7 +47,7 @@ static void adc_worker(uv_work_t* work_req) {
   }
 }
 
-JS_FUNCTION(AdcCons) {
+JS_FUNCTION(adc_onstructor) {
```

Comment:
Please correct the typos.(adc_onstructor ->adc_constructor)

Reply:
Good catch. I've fixed it.